### ISSUE TITLE
consulctl panic when using check register without argument

### ISSUE THREAD

**Issue by JrCs** [colebrumley/consulctl#1]
Hi Cole,

when running consulctl with check register argument (without other parameters), it's panic:

``` bash
./consulctl-linux-amd64 -d mydc -a "http://127.0.0.1:8500" check register
register flag redefined: h
panic: register flag redefined: h

goroutine 1 [running]:
flag.(*FlagSet).Var(0xc20806a180, 0x7efc1f85bfd0, 0xc208031b75, 0x78eb16, 0x1, 0x7aa7f0, 0x9)
    /usr/src/go/src/flag/flag.go:679 +0x494
flag.(*FlagSet).BoolVar(0xc20806a180, 0xc208031b75, 0x78eb16, 0x1, 0x0, 0x7aa7f0, 0x9)
    /usr/src/go/src/flag/flag.go:452 +0x8e
...
```

PS: what  command/container are you using to build this binary ?

**Comment by colebrumley** [comment]
This is one of those projects that I uploaded then immediately discovered already existed (https://github.com/CiscoCloud/consul-cli), so I haven't put any time into it since that initial push. At the time I just built it with `go get ./... && go build`, which still appears to work on my go 1.6 installation.

I'm not surprised that you got that panic, since IIRC several of the API functions were not really fleshed out when I uploaded the project.

It's probably worth a note in the README saying that this is here for posterity and that `consul-cli` is what you want to actually use :wink:
